uint16 cpu_load
float32 voltage